notebooks/08B_modeling_parallel_sorting_data.ipynb

```diff
@@ -264,9 +341,17 @@
     "        model_results = pd.concat([model_results, df])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Visualize Model Performance\n",
+    "With the `_plot_bars()` function, we can visualize the per-fold accuracy of each classifier for each model performance statistic. We notice that although performance varies slightly across folds, the tree models perform the best on performance data."
```

Comment:
It actually looks like the decision tree is the worst in every single fold and random forest and SVM vary between which is best, should be more specific here.

Reply:
7e92e1a0